src/select/mini.rs

```diff
@@ -381,9 +381,10 @@ where
                 format!("width: {w}px;")
             }
         });
-        let style_height = ctx.props()
-                .top_height
-                .map_or_else(String::new, |h| format!("height: {h}px;"));
+        let style_height = ctx
+            .props()
+            .top_height
+            .map_or_else(String::new, |h| format!("height: {h}px;"));
```

Comment:
Why is this rearranged?

Reply:
I added a note for that in the description. It seems like the ci.yml file was not configured correctly. I made an issue #148, and that is handled in #149

Reply:
I wrote why in https://github.com/aicers/frontary/issues/148.